apidoc/.gitignore

```diff
@@ -49,5 +49,4 @@
 /tmpl/*.bak
 /tmpl/ostree-unused.sgml
 /xml
-/version.xml
```

Comment:
Were these changes intentional? I can never tell WTF is going on with this file, but usually this happens for me when I don't configure with `--enable-gtk-doc`.